client/paperflow-web-v1/src/components/Network/Tooltip/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import styles from './Styles.module.scss';
+
+const App = ({ tooltipRef }) => (
```

Comment:
change the name of the functional component to something like `NetworkTooltip` from `App`